src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ApiShould.cs

```diff
@@ -2806,6 +2856,93 @@ private void CleanUpTable(string connString, string tableName)
             }
         }
 
+        /// <summary>
+        /// Creates a table with the specified number of bit columns.
+        /// </summary>
+        /// <param name="connString">The connection string to the database</param>
+        /// <param name="tableName">The table name</param>
+        /// <param name="columnsCount">The number of bit columns</param>
+        private void CreateTable(string connString, string tableName, int columnsCount)
+        {
+            using (var sqlConnection = new SqlConnection(connString))
+            {
+                sqlConnection.Open();
+
+                SqlCommand cmd = new SqlCommand(GenerateCreateQuery(tableName, columnsCount), sqlConnection);
+                cmd.ExecuteNonQuery();
+            }
+        }
+
+        /// <summary>
+        /// Drops the table if the specified table exists
+        /// </summary>
+        /// <param name="connString">The connection string to the database</param>
+        /// <param name="tableName">The name of the table to be dropped</param>
+        private void DropTableIfExists(string connString, string tableName)
+        {
+            using (var sqlConnection = new SqlConnection(connString))
+            {
+                sqlConnection.Open();
+                SqlCommand cmd = new SqlCommand($"DROP TABLE IF EXISTS {tableName};", sqlConnection);
+                cmd.ExecuteNonQuery();
+            }
+        }
```

Comment:
Using data utilities.
```suggestion
        private static void DropTableIfExists(string connString, string tableName)
        {
            using var sqlConnection = new SqlConnection(connString);
            sqlConnection.Open();
            DataTestUtility.DropTable(sqlConnection, tableName);
        }
```